native/general_double.go

```diff
@@ -10,6 +10,21 @@ import (
 	"math"
 )
 
+func checkMatrix(m, n int, a []float64, lda int) {
```

Comment:
You're welcome to rename it to `single_precision.bash` (make sure the go generate invocation is adjusted).